kinder/ci/tools/update-workflows/templates/workflows/control-plane-local-kubelet-mode-tasks.yaml

```diff
@@ -155,22 +157,21 @@ tasks:
 - name: post-upgrade
   description: |
     Run commands after kubeadm upgrade is called on all joined CP node to checks if
-    the kubelet's kubeconfig file points to the local apiserver.
+    the kubelet's kubeconfig file points to the remote apiserver.
   cmd: /bin/bash
   args:
     - -c
     - |
       set -x
 
-      IP_ADDRESS="$(docker inspect --format='{{ "{{" }} .NetworkSettings.IPAddress {{ "}}" }}' {{ .vars.clusterName }}-control-plane-1)"
+      IP_ADDRESS="$(docker inspect --format='{{ "{{" }} .NetworkSettings.IPAddress {{ "}}" }}' {{ .vars.clusterName }}-lb)"
+
```

Comment:
```
++ docker inspect '--format={{ .NetworkSettings.IPAddress }}' kinder-control-plane-local-kubelet-mode-lb
+ IP_ADDRESS=172.17.0.7
+ CMD='docker exec kinder-control-plane-local-kubelet-mode-control-plane-1'
+ docker exec kinder-control-plane-local-kubelet-mode-control-plane-1 grep 'server: https://172.17.0.7:6443' /etc/kubernetes/kubelet.conf
+ exit 1
```

Failed here.

Reply:
That's expected, it needs a commit from https://github.com/kubernetes/kubernetes/pull/129956 to succeed.